evm/src/test/java/org/hyperledger/besu/evm/code/CodeFactoryTest.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright contributors to Hyperledger Besu
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+package org.hyperledger.besu.evm.code;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.hyperledger.besu.datatypes.Hash;
+import org.hyperledger.besu.evm.Code;
+
+import java.security.Security;
+
+import org.apache.tuweni.bytes.Bytes;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CodeFactoryTest {
+
+  @BeforeAll
+  static void bcInit() {
+    Security.addProvider(new BouncyCastleProvider());
```

Comment:
Why do we need to init the security provider?  I'm not seeing how validating code would trip any crypto operations.

Reply:
it wasn't needed at all. I just removed it. It was a leftover from some strange behavior during a debug session.